pihole

```diff
@@ -180,7 +185,7 @@ helpFunc() {
 ::: Control all PiHole specific functions!
 :::
 ::: Usage: pihole [options]
-:::		Add -h after -w (whitelist), -b (blacklist), or -c (chronometer)  for more information on usage
+:::		Add -h after -w (whitelist), -b (blacklist), -c (chronometer), or -web (webpage)  for more information on usage
```

Comment:
Sure, do what you think is helps to improve this! Sometimes I pay too little attention on easily understandable documentation...